### DISCUSSION TITLE
Github Desktop infinite loading on fetch

### DISCUSSION THREAD

**Discussion by JacoboGanon** [community/community#73360] [Pull Requests]
### Select Topic Area

Bug

### Body

I downloaded github desktop a week or two ago, and it was fetching properly. However, now every time I attempt to fetch it will just say hang on while fetching infinitely.

**Comment by b8zeek** [comment]
This issue could occur because of number of reasons such as network issues, GitHub service disruptions, or problems with GitHub Desktop itself. Here are a few steps you can try to resolve this issue:
- make sure you have a stable internet connection, try to access other websites or services to confirm this
- check the GitHub status page to see if there are any ongoing issues with GitHub services
- close GitHub Desktop and reopen it, this can often resolve temporary issues
- make sure you are using the latest version of GitHub Desktop, you can check for updates in the application's settings
- try logging out of your GitHub account in GitHub Desktop and then logging back in
- you can clear the GitHub Desktop cache by going to `File` > `Options` > `Advanced` > `Reset Cache`
- you can try deleting the local repository (make sure to backup any uncommitted changes) and re-cloning it using GitHub Desktop

Anytime, you can reach out to GitHub support or check the GitHub Desktop issue tracker on GitHub for similar issues.

I am sure some of these will resolve your issue! Take care. 🍺 

🍀
